src/App.tsx

```diff
@@ -1,9 +1,35 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const insensitiveQuery = query.toLowerCase().trim();
```

Comment:
```suggestion
  const insensitiveQuery = query.toLowerCase():
```
test fails because of trim() method i guess. And better embed this variable in   copmairingItems  function, bellow